www/rustup.css

```diff
@@ -111,29 +130,28 @@ hr {
 #platform-instructions-win-arm64 > p,
 #platform-instructions-default > p,
 #platform-instructions-unknown > p {
-    width: 40rem;
+    width: 80%;
 }
 
 .rustup-command::before {
-    color: #999;
+    color: black;
     content: " $ ";
 	margin-left: 15px;
 }
 
 .rustup-command {
-    background-color: #515151;
```

Comment:
The white box looks more modern to me.

Reply:
I don't know, none of the color changes in this commit seem particularly well-motivated to me. Are they related/helpful/necessary with the other changes in this PR?

Reply:
It is just my taste. I think the white box looks more modern to me.
<img width="1438" alt="image" src="https://github.com/user-attachments/assets/71592e6a-b999-46db-8ac8-70004b880f49">
vs
![image](https://github.com/user-attachments/assets/587bdaf7-d192-4de9-a1e1-ad99bca92f28)

Reply:
I am OK to drop this change. Please let me know if you want to keep the original style.

Reply:
Okay, let's keep it.